Exam_Application/models/exam.py

```diff
@@ -0,0 +1,208 @@
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+
+
+class Examdetails(models.Model):
+    _name = "exam.details"
+    _description = "Exam Informations"
+    _rec_name = "seating_option"
+
+    exam_sequence_number = fields.Char(
+        string="Exam sequence",
+        required=True,
+        readonly=True,
+        copy=False,
+        default="New Sequence",
+    )
+
+    # --------
+    # sequence number generate
+    # --------
+    @api.model
+    def create(self, vals):
```

Comment:
Missing Doc string and inline comments inside the method.
Do not write any comment above the py method.